examples/README.md

```diff
@@ -16,5 +16,6 @@ This folder contains different examples of AMQ Streams custom resources and demo
     * Prometheus Alert
     * Sample Prometheus installation files
     * Sample Grafana installation files
+    * JMX Trans deployment
```

Comment:
we don't support JMX trans in AMQ Streams, or?

Reply:
No, we don't

Reply:
Fixed in most recent commit